www/plugin.d.ts

```diff
@@ -578,7 +578,7 @@ declare class Purchases {
      * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults
      * suite, otherwise it will use standardUserDefaults. Default is null, which will make the SDK use standardUserDefaults.
      */
-    static configure(apiKey: string, appUserID?: string | null, observerMode?: boolean, userDefaultsSuiteName?: string): void;
+    static configure(apiKey: string, callback: () => void, appUserID?: string | null, observerMode?: boolean, userDefaultsSuiteName?: string): void;
```

Comment:
Hmm should we have the `callback` as the last parameter? I'm not sure of the convention here but it's the convention in many languages, so I wanted to check.

Reply:
I think the reason is that the other parameters are optional. I am not sure though

Reply:
Yeah, it's because the others are optional. I also considered making the callback not required, so it's the last thing and it's not a breaking change. 
I went with this one because we already renamed `setup` -> `configure` so it's a breaking change regardless